docs/api.md

```diff
@@ -1067,27 +1298,88 @@ Example Response Body:
+| position        | The balance in position token (e.g. PBTC).                                     |
+| cachedMargin    | This is calculated as (index value - global index value)*position              |
+| pendingMargin   | This is the new (pending) value of the margin when a fill is still pending     |
+| pendingPosition | This is the new (pending) value of the position when a fill is still pending   |
+| indexValue      | The value of the global index from the last interaction with the account       |
+| indexTimestamp  | The timestamp when the index value was set                                     |
+
+
+### GET `/v1/perpetual-accounts`
+
+Description:
+This endpoint returns balances for all perpetual accounts.
+
+Query Params:
+| Field Name     | Description                                                     |
+|----------------|-----------------------------------------------------------------|
+| isLiquidatable | If set to true, returns accounts that are yet to be liquidated  |
+
+Example Response Body:
+
+```json
+{
+  "accounts": [
+    {
+      "owner": "0x000f7f22bfc28d940d4b68e13213ab17cf107790",
+      "market": "PBTC-USDC",
+      "orderNumber": "1011196300330012",
+      "cachedMargin": "36759577102",
+      "margin": "36759577102",
+      "position": "1201838526",
+      "cachedIndexValue": "1.211642619304704688",
+      "indexTimestamp": "1590100515"
+    },
+    {
+      "owner": "0x003f480be5b68c5e863d2990d043f3b58f64473f",
+      "market": "PBTC-USDC",
+      "orderNumber": "1010984800560005",
+      "cachedMargin": "368456014",
+      "margin": "368468282",
+      "position": "-3000000",
+      "cachedIndexValue": "1.207553221962107213",
+      "indexTimestamp": "1590072186"
+    }
+  ]
+}
+```
 
-| Field Name | Description                                         |
-|------------|-----------------------------------------------------|
-| owner      | The user's wallet address.                          |
-| balances   | An object with the user's balances for each market. |
-| margin     | The balance in settlement token (e.g. USDC).        |
-| position   | The balance in position token (e.g. PBTC).          |
+#### Account Response Object
+
+| Field Name      | Description                                                                    |
+|-----------------|--------------------------------------------------------------------------------|
+| owner           | The user's wallet address.                                                     |
+| balances        | An object with the user's balances for each market.                            |
+| margin          | The balance in settlement token (e.g. USDC).                                   |
+| position        | The balance in position token (e.g. PBTC).                                     |
+| cachedMargin    | This is calculated as (index value - global index value)*position              |
```

Comment:
This is not calculated like that. This is the last `margin` on-chain stored in brokerage. We use this value to calculate the margin

Reply:
fixed